### PR TITLE
SEQNG-613: GPI Keywords

### PR DESCRIPTION
**Pull request by cquiroz**
This PR does the bulk of the work to get keywords onto GPI images. It has been tested against the instrument and it gets the keywords written except for the following exceptions:

* CRFOLLOW which seems to be not in use. we'll follow up with science
* Keywords from gcal/gws. They need to be modified to work with either the DHS or GDS

### CODE REVIEW

File: modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala [hunk 1]
```diff
@@ -33,22 +34,19 @@ final case class GDSClient(client: Client[IO], gdsUri: Uri)
...
-              </data>
-            </array>
-          </value>
-        </param>
+        {keywordsParam(ks)}
       </params>
     </methodCall>
 
+  private def handleConnectionError
+    : PartialFunction[Throwable, SeqexecFailure] = {
```
Comment (tpolecat) [inline modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: Why is this a `PartialFunction`?

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: We pass it to `EitherT.leftMap` which takes that signature

Comment (tpolecat) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: It does? Weird.

Comment (tpolecat) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/EitherT.scala#L109

?

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: Oh, i got confused with the transformers i guess when trying to disentangle the monad transformers stack

Comment (tpolecat) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: I would 👍 a PR to remove all the type aliases ;-)

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: yeah, maybe I can come up with some magic sed command to do it

Comment (cquiroz) [reply modules/seqexec/server/src/main/scala/seqexec/server/keywords/GDSClient.scala]: Fixed the signature

**Pull request merged**